astropy/utils/iers/tests/test_leap_second.py

```diff
@@ -9,14 +10,14 @@
 import pytest
 from numpy.testing import assert_array_equal
 
-from astropy.tests.tests.test_imports import test_imports
 from astropy.time import Time, TimeDelta
 from astropy.utils.data import get_pkg_data_filename
 from astropy.utils.iers import iers
 
-# Import every top-level astropy module as a test that the ERFA leap second
+# Import every astropy module as a test that the ERFA leap second
 # table is not updated for normal imports.
-test_imports()
+for finder, name, _ in pkgutil.walk_packages(["astropy"], prefix="astropy."):
```

Comment:
If I try this on the command line, the iteration is empty - see above.

Another reason perhaps not to fiddle with it in this PR...